src/preview.ts

```diff
@@ -25,9 +25,16 @@ function preview(editor: TextEditor) {
 
     let doc = editor.document;
     if (doc != currentDoc) {
-        commands.executeCommand('markdown.showPreviewToSide').then(() => {
-            commands.executeCommand('workbench.action.navigateBack');
-        });
+        try {
+            commands.executeCommand("markdown-preview-enhanced.openPreview").then(() => {
```

Comment:
Is `markdown-preview-enhanced.openPreview` a built-in command of vscode? I don't have this command in my vscode 🤔

Reply:
No, it just an extension that supports Pandoc Markdown.
I think I should keep it for myself.
I will remove from the pull request.